packages/app/components/Today/ReleaseNotes.tsx

```diff
@@ -1,45 +1,24 @@
 import { Modal, notification } from "antd";
 import { ReactElement, useState, useEffect } from "react";
 import { useLocalStorage } from "../../hooks/useLocalStorage";
-import { NotionRenderer } from "react-notion";
+import { NotionRenderer, BlockMapType } from "react-notion";
+import { API } from "@koh/api-client";
 
 export default function ReleaseNotes(): ReactElement {
   const [releaseNotesLastSeen, setReleaseNotesLastSeen] = useLocalStorage(
     "releaseNotesLastSeen",
     null
   );
   const [releaseNotesLastUpdated, setReleaseNotesLastUpdated] = useState(0);
-  const [notionReleaseNotes, setNotionReleaseNotes] = useState();
+  const [notionReleaseNotes, setNotionReleaseNotes] = useState<BlockMapType>();
   const [showReleaseNotes, setShowReleaseNotes] = useState(false);
 
   useEffect(() => {
-    fetch(
-      "https://notion-api.splitbee.io/v1/page/abba246bfa0847baa2706ab30d0c6c7d"
-    )
-      .then((res) => res.json())
-      .then(
-        (result) => {
-          try {
-            const timeText =
-              result["beae2a02-249e-4b61-9bfc-81258d93f20d"]?.value?.properties
-                ?.title[0][0];
-            setReleaseNotesLastUpdated(timeText.split("Unix ")[1] * 1000);
-          } catch (e) {
-            console.log("Error Parsing release notes time:", e);
-          }
-          // Remove the time block and page link block from page
-          result[
-            "beae2a02-249e-4b61-9bfc-81258d93f20d"
-          ].value.properties.title = [];
-          result[
-            "4d25f393-e570-4cd5-ad66-b278a0924225"
-          ].value.properties.title = [];
-          setNotionReleaseNotes(result);
-        },
-        (error) => {
-          console.log("Error fetching release notes", error);
-        }
-      );
+    (async () => {
+      const data = await API.releaseNotes.get()
+      setNotionReleaseNotes(data.releaseNotes as BlockMapType)
```

Comment:
wait why not `useSWR`

Reply:
It's been a minute since I used it so I totally forgot 🤦 

I'll fix that up right away

Reply:
Fixed.